bayesian-bandit.js

```diff
@@ -6,7 +6,10 @@
   // Arm - one of the multi-armed bandit's arms, tracking observed rewards. //
   ////////////////////////////////////////////////////////////////////////////
 
-  function Arm() { }
+  function Arm(args) {
+    this.count = args && args.count || 0;
+    this.sum = args && args.sum || 0;
```

Comment:
Now that we have this maybe we can remove the `Arm.prototype.count = 0` and `Arm.prototype.sum = 0` lines?